app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= stylesheet_link_tag 'signIn' %>
+<%= stylesheet_link_tag 'category_index' %>
+
+<div class="cat_header">
+ <div class="i_back"> <%= link_to "<", category_path(@category), class:"back" %></div>
+ <div class="p_title">New Transaction</div>
+ <div class="i_menu"><img src="https://icon-library.com/images/white-search-icon-png/white-search-icon-png-4.jpg" alt="No transactions_img" style="width: 150px;"></div>
+</div>
```

Comment:
- [Optional] Dear @fredo509, I notice you have removed the option for selecting the category while creating a transaction. It makes sense in some cases and I totally understand your point of view. But as professional programmers, we need to follow the requirements, and the requirements state to have a checkbox for selecting two or more categories for a single transaction, (`categories (mandatory at least one)`). Please consider following the requirements closely in the future. 

### Your form 
![Screen Shot 2023-08-19 at 10 04 50 AM](https://github.com/fredo509/Budget_App/assets/69077061/865aaa3e-3189-467c-a1cc-b3876048faf6)